### ISSUE TITLE
Implementare dei Test con Selenium

### ISSUE THREAD

**Issue by francescomalatesta** [laravel-italia/site#2]
Implementare dei test sull'interfaccia e su tutti i vari casi d'uso con Selenium potrebbe essere una buona idea. Non soltanto per rafforzare il codice del sito ma anche per poi scriverci una mini-serie di articoli.

**Comment by ludo237** [comment]
Non serve selenium, basta anche il toolset integrato di laravel per fare acceptance

**Comment by francescomalatesta** [comment]
Yep, alcune cose realizzate con js tuttavia non sarebbero testabili (modal per la pubblicazione). Certo si tratta comunque di pochissime cose :)
